src/test/java/util/HttpRequestUtilsTest.java

```diff
@@ -71,6 +44,34 @@ void getStartLine() {
         assertThat(param).isEqualTo(expectedStartLine);
     }
 
+    @Test
+    @DisplayName("getMethod 메서드 테스트: HTTP 요청 라인에서 HTTP 메소드가 정확하게 추출되는지 검증")
+    void getMethod() {
+        // given
+        String line = "GET /index.html HTTP/1.1";
+        String expectedMethod = "GET";
+
+        // when
+        String param = HttpRequestUtils.getMethod(line);
+
+        // then
+        assertThat(param).isEqualTo(expectedMethod);
```

Comment:
```suggestion
        assertThat(param).isEqualTo("GET");
```
이거는 이렇게 직접 값을 넣는게 더 좋아 보이는데 어떠세요?